ci/triage_build.sh

```diff
@@ -20,8 +20,8 @@ KIND="$RUNNER_OS $ARCH"
 
 # If it's a scheduled build or [pip-pre] in commit message, use pip-pre
 if [[ "$GITHUB_EVENT_NAME" == "schedule" ]] || [[ "$MSG" = *'[pip-pre]'* ]]; then
-    echo "Using NumPy pip-pre wheel and, setting CIBW_BEFORE_BUILD, CIBW_BUILD_FRONTEND and CIBW_ENABLE"
-    echo "CIBW_BEFORE_BUILD=pip install --pre --only-binary numpy --extra-index-url https://pypi.anaconda.org/scientific-python-nightly-wheels/simple \"numpy>=2.0.0.dev0\" \"Cython>=0.29.31,<4\" pkgconfig \"setuptools>=77\" wheel" | tee -a $GITHUB_ENV
+    echo "Using NumPy pip-pre wheel; setting CIBW_BEFORE_BUILD and CIBW_BUILD_FRONTEND"
```

Comment:
we need to keep this in sync with the `[build-system]` table in `pyproject.toml`. Could you also set an explicit lower bound for `packaging` while you're at it ? (using a recent version is fine)